content/en/docs/release-oversight/disruption-testing/aggregated-disruption.md

```diff
@@ -0,0 +1,133 @@
+ [sig-api-machinery] disruption/kube-api connection/reused should be available throughout the test
+ [sig-api-machinery] disruption/oauth-api connection/new should be available throughout the test
+ [sig-api-machinery] disruption/oauth-api connection/reused should be available throughout the test
+ [sig-api-machinery] disruption/openshift-api connection/new should be available throughout the test
+ [sig-api-machinery] disruption/openshift-api connection/reused should be available throughout the test
+ [sig-network-edge] ns/openshift-authentication route/oauth-openshift disruption/ingress-to-oauth-server connection/new should be available throughout the test
+ [sig-network-edge] ns/openshift-authentication route/oauth-openshift disruption/ingress-to-oauth-server connection/reused should be available throughout the test
+ [sig-network-edge] ns/openshift-console route/console disruption/ingress-to-console connection/new should be available throughout the test
+ [sig-network-edge] ns/openshift-console route/console disruption/ingress-to-console connection/reused should be available throughout the test
+ [sig-api-machinery] disruption/cache-kube-api connection/new should be available throughout the test
+ [sig-api-machinery] disruption/cache-kube-api connection/reused should be available throughout the test
+ [sig-api-machinery] disruption/cache-oauth-api connection/new should be available throughout the test
+ [sig-api-machinery] disruption/cache-oauth-api connection/reused should be available throughout the test
+ [sig-api-machinery] disruption/cache-openshift-api connection/new should be available throughout the test
+ [sig-api-machinery] disruption/cache-openshift-api connection/reused should be available throughout the test
+ [sig-trt] disruption/ci-cluster-network-liveness connection/new should be available throughout the test
+ [sig-trt] disruption/ci-cluster-network-liveness connection/reused should be available throughout the test
+```
+
+For these tests, disruption tests are aggregated in a way similar to [how the other junit tests are aggregated](../../improving-ci-signal).
+
+2.  The second type consists of tests that look like these examples below and are defined in [openshift/ci-tools repo](https://github.com/openshift/ci-tools/blob/d540ac734fd1b6bbdc05811c238e362cda2286c1/pkg/jobrunaggregator/jobrunaggregatoranalyzer/analyzer_disruption.go#L57-L64):
+
+```html
+ cache-kube-api-new-connections disruption P70 should not be worse
+ cache-kube-api-new-connections disruption P85 should not be worse
+ cache-openshift-api-new-connections disruption P95 should not be worse
+ cache-kube-api-new-connections mean disruption should be less than historical plus five standard deviations
+ cache-kube-api-new-connections zero-disruption should not be worse
+```
+
+These tests are aggregated similar to the first type of disruption tests.  For each backend, we have a
```

Comment:
We should have a brief introduction to the mechanism of this aggregation. Something like "instead of simply aggregating test results (success and failures) across all the aggregated job runs, this group collects the disruption numbers from all the job runs and do some comparisons with historical data for each job run. The comparisons will result in a status (success or failure) for each of the job run. These status will be aggregated similar to regular aggregation. 

There are three types of comparisons: Mean disruption check, Percentile disruption check and Zero disruption check:".